org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/engine/connected/ConnectedEngineFacade.java

```diff
@@ -234,7 +234,8 @@ public String getServerVersion() {
     if (getStorageState() != State.UPDATED) {
       return NEED_UPDATE;
     }
-    return updateStatus.getServerVersion();
+    String version = updateStatus.getServerVersion();
```

Comment:
Here, because of the previous condition, we know that storage state == State.UPDATED, and so getServerVersion() should be non null.

Reply:
I made this change because the `GlobalStorageStatus.getServerVersion()` method is annotated with `@CheckForNull` and I got an issue, that's why I added the check. So we can just ignore the issue ?

Reply:
Yes, I usually mark it as FP in this case.